cluster/gce/gci/configure-helper.sh

```diff
@@ -586,7 +586,7 @@ function mount-master-pd {
 
   # Contains all the data stored in etcd.
   mkdir -p "${mount_point}/var/etcd"
-  chmod 700 "${mount_point}/var/etcd"
+  chmod 750 "${mount_point}/var/etcd"
```

Comment:
I'm not the expert to tell if it's safe to allow group reading the data. The rest of the changes LGTM.